features/main/input_output.feature

```diff
@@ -287,3 +287,30 @@ Feature: DTO input and output
       "data": 123
     }
     """
+
+  @!mongodb
```

Comment:
Why?

Reply:
Because I don't want to create a document for this and also because this entity isn't even tight to a database persister so there's no point of running it with mongodb.

Reply:
Yes but I think it would be better to have it working with MongoDB anyway. This tag is only here when the functionality is not compatible with MongoDB.